packages/database/src/entity/index.ts

```diff
@@ -1,4 +1,5 @@
-export { Degree } from './Degree'
 export { Module } from './Module'
```

Comment:
This might solve the undefined metadata issue, as it appears the exports need to be in topological sort order (dependency order)